resources/js/nftLoader/nftLoader.js

```diff
@@ -391,10 +391,10 @@
   }
 }
 
-  async function jsonParser(requestURL, callback) {
```

Comment:
I had to remove the async/await  and ES6 syntax for the uglyfier, Anyway had to use the command `grunt uglify --force` . i need to find a better solution.

Reply:
this was solved with https://github.com/kalwalt/kalwalt-interactivity-AR/pull/18/commits/0fa1572bacf11784d47f298b4e95fe107c712481